pymc/logprob/order.py

```diff
@@ -254,3 +334,23 @@ def max_neg_logprob(op, values, base_var, **kwargs):
     logprob = (n - 1) * pt.math.log(1 - pt.math.exp(logcdf)) + logprob + pt.math.log(n)
 
     return logprob
+
+
+@_logprob.register(MeasurableMaxNegDiscrete)
+def maxneg_logprob_discrete(op, values, base_rv, **kwargs):
+    r"""Compute the log-likelihood graph for the `Max` operation.
+
+    The formula that we use here is :
+    .. math::
+        \ln(P_{(n)}(x)) = \ln(F(x)^n - F(x-1)^n)
```

Comment:
Needs to be edited?

Reply:
```suggestion
        \ln(P_{(n)}(x)) = \ln((1 - F(x - 1))^n - (1 - F(x))^n)
```
We are working with the survival function here #6959